packages/core/src/App/Components/Layout/Header/menu-links.jsx

```diff
@@ -92,13 +92,14 @@ const MenuLinks = observer(({ is_traders_hub_routes = false }) => {
     const { client, ui } = useStore();
     const { is_logged_in } = client;
     const { is_mobile } = ui;
+    const { is_next_wallet_enabled } = useFeatureFlags();
 
     if (!is_logged_in) return <></>;
 
     return (
         <div key={`menu-links__${i18n.language}`} className='header__menu-links'>
             {!is_traders_hub_routes && <ReportTab />}
-            {!is_mobile && <CashierTab />}
+            {!is_mobile && !is_next_wallet_enabled && <CashierTab />}
```

Comment:
We need this check for now. Will remove it once we are not dependant on feature flags.